docs/applications/containers/troubleshooting-kubernetes/index.md

```diff
@@ -120,21 +127,29 @@ Enter `exit` to later leave this shell.
 
 ### Viewing Master and Worker Logs
 
-The locations of logs on your master nodes are:
+If you have set up Kubernetes using a method that does not include [**systemd**](https://linode.com/docs/quick-answers/linux-essentials/what-is-systemd/), the location of logs on your master nodes are in most cases as follows:
 
 - `/var/log/kube-apiserver.log` - [API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
 - `/var/log/kube-scheduler.log` - [Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/ )
 - `/var/log/kube-controller-manager.log` - [Replication controller manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/)
 
-    {{< note >}}
-You will not be able to directly access your master nodes if you are using Linode's LKE service.
-{{< /note >}}
-
 On your worker nodes:
 
 - `/var/log/kubelet.log` - [Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
 - `/var/log/kube-proxy.log` - [Kube proxy](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/)
 
+If you do have systemd implemented, then the above logs will not be accessible and you'll need to access kubelet logs using `journalctl` instead. Below is an example command for accessing kubelet logs through journalctl:
+
+    journalctl --unit kubelet
+
+In some cases, the information you find from kubelet may prompt you to look into individual container logs for more information. When using Docker as a container runtime, you can find information on all running containers with the `docker ps` command. For example, to find the container running your API server, enter the following:
+
+    docker ps | grep apiserver
+
+Then, using the container ID that appears, enter `docker logs` to view the logs for your API server:
```

Comment:
Going to add a bit more detail here and include an example of what the output of the docker ps | grep command is, where the container ID is in that output